_layouts/contributed-data.html

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
please delete the jekyll templates that are not used yet. We will copy them over as we get to each page. Unused code will just get stale and confusing.